qemu/tests/rebase_negative_test.py

```diff
@@ -0,0 +1,75 @@
+import re
+import logging
+
+from virttest import qemu_storage
+from virttest import error_context
+from virttest import data_dir
+
+from avocado.utils import process
+
+
+@error_context.context_aware
+def run(test, params, env):
+    """
+    qemu-img rebase negative test:
+    1) Create images for testing using avocado
+    2) Change the backing file of snapshot
+    3) Check result. Case passed only if expected rebase
+    opearations are failed and no core dump is generated
+
+    :param test: Qemu test object
+    :param params: Dictionary with the test parameters
+    :param env: Dictionary with test environment
+    """
+    rebase_chain = params.get("rebase_list", "").split(";")
+    error_context.context("Change the backing file of snapshot",
+                          logging.info)
+    for images in rebase_chain:
+        output = ""
+        cache_mode = params.get("cache_mode", None)
+        images = map(lambda x: x.strip(), images.split(">"))
```

Comment:
Suggest `re.split("\s*>\s*", images)`, otherwise, needs to wrap with `list()` to support python 3.